cli/cmd/root.go

```diff
@@ -56,4 +58,6 @@ func init() {
 	RootCmd.PersistentFlags().StringVarP(&url, "url", "u", "http://localhost:8083", "kafka connect URL")
 	RootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, `/!\ very verbose`)
 	RootCmd.PersistentFlags().BoolVarP(&SSLInsecure, "insecure-skip-verify", "i", false, `skip SSL/TLS verification`)
+	RootCmd.PersistentFlags().StringVarP(&SSLClientCertificate, "ssl-client-certificate", "C", "", `client certificate, must contain PEM encoded data`)
+	RootCmd.PersistentFlags().StringVarP(&SSLClientPrivateKey, "ssl-client-key", "K", "", `client private key`)
```

Comment:
If i understand this correctly, these parameters are files paths?

If yes, should add this info in the description.
e.g: `file path to client private key`